corehq/apps/cloudcare/templates/formplayer-common/base.html

```diff
@@ -33,10 +33,10 @@
   <link type="text/css"
         rel="stylesheet"
         media="screen"
-        href="{% static "jquery-ui/themes/redmond/jquery-ui.min.css" %}" />
+        href="{% static "jquery-ui-built-themes/redmond/jquery-ui.min.css" %}" />
```

Comment:
Can this just be removed? `FormplayerMain` uses `use_jquery_ui`.

Reply:
hmmm.... i feel like it's still using this?

Reply:
I had thought it was included twice, but looks like it's not. Carry on!